src/unity/python/turicreate/visualization/__init__.py

```diff
@@ -4,111 +4,35 @@
 # Use of this source code is governed by a BSD-3-clause license that can
 # be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
 """
-Turi Create Visualization allows you to create, view, and save Plot objects.
+Turi Create Visualization allows you to create, view, and save Plot objects. 
+The Plot class is used to create an immutable object representation of a 
+visualization and has the following methods:
+- `show`: A method for displaying the Plot object.
+- `save`: A method for saving the Plot object in a vega representation.
+
+Turi Create Visualization also allows you to call `show` 
+on data sources like SFrames and SArrays:
+
+- `show`: Plots the data in `x` on the X axis and the data in `y` on the Y axis
+          in a 2d visualization, and shows the resulting visualization.
+          Uses a heuristic to choose the visualization, described in the API 
+          documentation. 
+          This show method supports SArrays of dtypes: int, float, str.
+
+Furthermore, Turi Create provides functionality to create independent plots 
+like scatter plots, categorical heatmaps, box plots, heatmaps, 
+columnwise summaries, and item frequency plots using the following API:
+
+- scatter(x, y, xlabel, ylabel, title)
+- categorical_heatmap(x, y, xlabel, ylabel, title)
+- heatmap(x, y, xlabel, ylabel, title)
+- box_plot(x, y, xlabel, ylabel, title)
+- columnwise_summary(sf)
+- histogram(sa, xlabel, ylabel, title)
+- item_frequency(sa, xlabel, ylabel, title)
+
+To know more, please see the API docs at https://apple.github.io/turicreate/docs/api/
```

Comment:
I would remove this line, or rephrase it to reference the docstrings of the referenced classes/methods. Since the user is already in the UX flow of looking up docstrings on Python objects, they'll probably want to stay in that flow as they explore. How about "For more detailed information, please refer to the docstrings for each of those classes/methods."